### PR TITLE
fix: repo list no data state

### PR DESCRIPTION
**Pull request by shaurya-harness**
fixes : 
1. **commit changes onSuccess navigation**
2. **default branch for repo summary/files**
3. **commit changes to current branch**

4. **repo list no data state**
Screenshot 2024-12-11 at 11 34 37 AM

Screenshot 2024-12-11 at 11 34 49 AM

Screenshot 2024-12-11 at 11 34 57 AM

5. **PR list no data state**
Screenshot 2024-12-11 at 3 26 39 PM

Screenshot 2024-12-11 at 3 26 55 PM

6. **Monospace font for Commit Sha's across the pages**
Screenshot 2024-12-11 at 10 49 21 PM

Screenshot 2024-12-11 at 10 49 28 PM

Screenshot 2024-12-11 at 10 49 38 PM

Screenshot 2024-12-11 at 10 49 15 PM

### CODE REVIEW

File: packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx [hunk 1]
```diff
@@ -139,36 +148,45 @@ const PullRequestList: FC<PullRequestPageProps> = ({
...
-          secondaryButton={{
-            label: 'Clear filters',
-            onClick: filterHandlers.handleResetFilters
-          }}
-        />
+        <StackedList.Root>
+          <div className="flex min-h-[50vh] items-center justify-center py-20">
+            <NoData
+              iconName="no-search-magnifying-glass"
+              title="No search results"
```
Comment (cjlee01) [inline packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx:165]: add string to views.json

File: packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx [hunk 2]
```diff
@@ -139,36 +148,45 @@ const PullRequestList: FC<PullRequestPageProps> = ({
...
+              description={['Check your spelling and filter options,', 'or search for a different keyword.']}
```
Comment (cjlee01) [inline packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx:166]: add string to views.json

File: packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx [hunk 3]
```diff
@@ -139,36 +148,45 @@ const PullRequestList: FC<PullRequestPageProps> = ({
...
+              primaryButton={{
+                label: 'Clear search',
```
Comment (cjlee01) [inline packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx:168]: add string to views.json

File: packages/ui/src/views/repo/repo-list/repo-list.tsx [hunk 4]
```diff
@@ -55,41 +55,41 @@ export function RepoList({
...
-                onClick: () => {
-                  /* TODO: add create handler */
-                }
-              }}
-            />
-          )}
+          <NoData
+            iconName="no-search-magnifying-glass"
+            title="No search results"
+            description={['Check your spelling and filter options,', 'or search for a different keyword.']}
```
Comment (cjlee01) [inline packages/ui/src/views/repo/repo-list/repo-list.tsx:64]: add string to views.json

File: packages/ui/src/views/repo/repo-list/repo-list.tsx [hunk 5]
```diff
@@ -55,41 +55,41 @@ export function RepoList({
...
+            primaryButton={{
+              label: 'Clear search',
```
Comment (cjlee01) [inline packages/ui/src/views/repo/repo-list/repo-list.tsx:66]: add string to views.json

File: packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx (hunk 2 above)
Comment (shaurya-harness) [reply packages/ui/src/views/repo/pull-request/pull-request-list-page.tsx:166]: already there

File: packages/ui/src/views/repo/repo-list/repo-list.tsx (hunk 5 above)
Comment (shaurya-harness) [reply packages/ui/src/views/repo/repo-list/repo-list.tsx:66]: already there

File: packages/ui/src/views/repo/repo-list/repo-list.tsx (hunk 4 above)
Comment (shaurya-harness) [reply packages/ui/src/views/repo/repo-list/repo-list.tsx:64]: already there

**Review by cjlee01 — APPROVED** [review]

**Pull request merged**